script/Common.s.sol

```diff
@@ -340,6 +341,35 @@ abstract contract Common is Contracts, Params {
     rewardedActions = new RewardedActions();
   }
 
+  function _deployUniV3Pool() internal {
+    (address _token0, address _token1) = address(systemCoin) < address(collateral[WETH])
+      ? (address(systemCoin), address(collateral[WETH]))
+      : (address(collateral[WETH]), address(systemCoin));
+
+    uint256 _ethUSDPrice = delayedOracle[WETH].read();
+
+    address _uniV3Pool = INonfungiblePositionManager(NONFUNGIBLE_POSITION_MANAGER).createAndInitializePoolIfNecessary({
+      token0: _token0,
+      token1: _token1,
+      fee: HAI_POOL_FEE_TIER,
+      sqrtPriceX96: uint160(_ethUSDPrice) // REVIEW: How to calculate sqrtPriceX96 on-chain?
```

Comment:
what about https://etherscan.io/address/0x60594a405d53811d3bc4766596efd80fd545a270#readContract
😉 

NOTE: the initial price is not crucial

Reply:
I hardcoded the initial `sqrtPriceX96` at an exact 1:1800 quote.